fad.lisp

```diff
@@ -149,7 +148,7 @@ directory is returned as if by PATHNAME-AS-DIRECTORY."
                      directory-form)))
                (ignore-errors
                  (probe-file (pathname-as-file pathspec))))
-  #-(or :sbcl :cmu :scl :lispworks :openmcl :allegro :clisp :cormanlisp :ecl :abcl :digitool)
+  #-(or :sbcl :cmu :scl :lispworks :openmcl :allegro :clisp :cormanlisp :ecl :abcl :digitool :sbcl :clasp)
```

Comment:
:sbcl already in there  - at beginning of this line/statement. I only wanted to add :clasp here.